website/docs/guides/customize-schema-alias.md

```diff
@@ -0,0 +1,486 @@
+---
+title: Customize schema and alias
```

Comment:
This is not just schema and alias, this is database, schema and alias.

I would change the title to be more specific as well, like
"Customize dbt models database, schema and alias

Reply:
Thanks for this @b-per I'll get the title updated

Reply:
```suggestion
title: Customize dbt models database, schema and alias
```